.dockerfilelintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "skipShellCheck": false,
+  "skipCritical": false,
+  "disableLineBreaks": false
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project